View Controllers/TimelineViewController.m

```diff
@@ -64,13 +64,16 @@ - (void)fetchTweets {
         [self.refreshControl endRefreshing];
     }];
 }
-/*
-- (void)didTweet {
-   // [_arrayOfTweets insertObject:self atIndex:0];
+
+- (void)didTweet:(Tweet *)tweet{
+    if(!self.arrayOfTweets) {
```

Comment:
I didn't catch this earlier but `arrayOfTweets` should just be `tweets`. Anytime a model is used with plural it's generally assumed to be an array.